File: 63-spring-boot-mybatis-mini/src/main/java/com/gzz/service/UserService.java
package com.gzz.service;
/**
 * @author https://www.jianshu.com/u/3bd57d5f1074
 * @date 2019-12-24 14:50:00
 */
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.gzz.dao.UserDao;
import com.gzz.model.User;

@Service
public class UserService {

	@Autowired
	private UserDao dao;

	public int add(User user) {
		return dao.insertUser(user);
	}
}

File: 99-spring-boot-kotlin/src/main/java/com/gzz/mvc/UserService.kt
package com.gzz.mvc

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service
import com.gzz.base.Page

@Service
class UserService {
	@Autowired
	lateinit var dao: UserDao

	fun queryList(cond: UserCond): List<User> {
		return dao.queryList(cond);
	}

	fun queryPage(cond: UserCond): Page<User> {
		return dao.queryPage(cond);
	}
}